.github/workflows/test-typescript.yml

```diff
@@ -17,7 +17,7 @@ jobs:
 
     strategy:
       matrix:
-        typescript-version: [4.2, 4.3, 4.4, 4.5, 4.6, 4.7, 4.8, 4.9 ]
+        typescript-version: [4.2, 4.3, 4.4, 4.5, 4.6, 4.7, 4.8, 4.9, 5.0, 5.1, 5.2, 5.3, 5.4, 5.5, 5.6 ]
```

Comment:
Is this overkill? I definitely want to test 4.2 as we state in the docs that the minimum TS version is 4.2. Do we need to test all these versions. Any suggestions?

Reply:
Fine for now.